cpp/ps0003/p03.cpp

```diff
@@ -4,11 +4,17 @@ using namespace std;
 
 int main()
 {
-  int x=0;
-  cin>>x;
- if (x%2==0)
-    cout<< "Ever"<<endl;
- else
-     cout<< "Odd"<<endl;
+  // prompting user for number 2/2
+  int x = 0;
+  cin >> num;
 
+  // checking if the number is odd or even 3/3
+  cout << "Number is ";
+  if (num % 2 == 0)
+    cout << "even" << endl;
+  else
+    cout << "odd" << endl;

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console